i3_agenda/event.py

```diff
@@ -14,17 +18,17 @@ def __init__(
         summary: str,
         start_time: float,
         end_time: float,
-        location: str,
-    ):
+        location: Union[str, None]
```

Comment:
why?

Reply:
Because in `from_json` Event is sometime created with a `None` location, which seems legit to me.